build.gradle

```diff
@@ -1,7 +1,25 @@
-defaultTasks 'build'
+/*
+ * Copyright (C) 2018 Knot.x Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
 
-allprojects {
+plugins {
```

Comment:
Can we please move your changes to `gradle/knotx-common.gradle` file. Then you can reference this file in all required modules.

The example: https://github.com/Cognifide/gradle-aem-multi/blob/master/gradle/common.gradle.kts
https://github.com/Cognifide/gradle-aem-multi/blob/master/aem/build.gradle.kts#L35